pkg/controllers/binding/common.go

```diff
@@ -193,6 +197,23 @@ func RecordAppliedOverrides(cops *overridemanager.AppliedOverrides, ops *overrid
 	return annotations, nil
 }
 
+// parseConflictResolution determine the conflictResolution annotation of Work: preferentially inherit from RT, then RB
+func parseConflictResolution(workload *unstructured.Unstructured, conflictResolutionInBinding policyv1alpha1.ConflictResolution,
```

Comment:
```suggestion
func mergeConflictResolution(workload *unstructured.Unstructured, conflictResolutionInBinding policyv1alpha1.ConflictResolution,
```
How about replacing `parse` to `merge`?